lib/pengine/bundle.c

```diff
@@ -1491,10 +1491,19 @@ pe__bundle_text(pcmk__output_t *out, va_list args)
     GList *only_node = va_arg(args, GList *);
     GList *only_rsc = va_arg(args, GList *);
 
+    const char *desc = NULL;
+
     pe__bundle_variant_data_t *bundle_data = NULL;
     int rc = pcmk_rc_no_output;
     gboolean print_everything = TRUE;
 
+    // User-supplied description
+    if (pcmk_is_set(show_opts, pcmk_show_rsc_only)
+        || pcmk__list_of_multiple(rsc->running_on)) {
+        desc = crm_element_value(rsc->xml, XML_ATTR_DESC);
+    }
```

Comment:
To make it easier, it would probably be worthwhile to functionize this whole block as something like

    desc = pe__resource_description(rsc, show_opts);

That would probably go best in pe_output.c.